charts/spire/charts/spire-server/values.yaml

```diff
@@ -217,3 +223,11 @@ nodeAttestor:
   k8sPsat:
     enabled: true
     serviceAccountAllowList: []
+
+# FIXME Not implemented yet
+postgresql:
+  enabled: false
+
+# FIXME Not implemented yet
+mysql:
+  enabled: false
```

Comment:
I think this should probably go under the persistence of datastore sections

Reply:
Thanks for the review.

Unfortunately, a nested chart can't contain any nestings in the values file of the chart its nested into.
so can be anything at the root of the chart, such as:
```
postgresql:
  enabled: true
```

but cant be like:
```yaml
dataStore:
  sql:
     postgresql:
       enabled: true
```

Its a limitation of helm itself.